lark/tree.pyi

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+from typing import List, Callable, Iterator, Union, Optional
+from .lexer import Token
+
+
+class Tree:
+
+    data: str
+    children: List[Union[str, Tree]]
+    meta: Token
+
+    def __init__(self, data: str, children: List[Tree], meta: Optional[Token] = None):
+        ...
+
+    def pretty(self, indent_str: str = ...) -> str:
+        ...
+
+    def find_pred(self, pred: Callable[[Tree], bool]) -> Iterator[Tree]:
+        ...
+
+    def find_data(self, data: str) -> Iterator[Tree]:
+        ...
+
+    def iter_subtrees(self) -> Iterator[Tree]:
+        ...
+
+    def iter_subtrees_topdown(self) -> Iterator[Tree]:
+        ...
+
+    def __eq__(self, other: object) -> bool:
```

Comment:
Why is this necessary? No one calls `__eq__` directly. Perhaps there's a reason I'm not aware of?

Reply:
Well, in this case it's unnecessary. Or represent a override method?

Reply:
What do you mean?

Reply:
Since superclass `object` already have it, unless it's a abstract method, I will remove it.